experiments.qmd

```diff
@@ -6,36 +6,40 @@ This section is primarily written for project leaders, so lab assistants can ski
 
 ### Finding lab assistant availability 
-1. Have all lab assistants block out their available times on the cchil.unl@gmail.com Google calendar [Dog lab availability](https://calendar.google.com/calendar/u/0?cid=OXJsaHQ5NTAzdnJvOGE2Njk3b2M4NjBjcThAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ). 
-1. Look for blocks of time in which at least two lab assistants are available. These will be the "open" times on the Calendly Calendar in which owners can sign up for appointment slots. 
+1. Have all lab assistants block out their available times on the cchil.unl@gmail.com Google calendar [Dog lab availability](https://calendar.google.com/calendar/u/0?cid=OXJsaHQ5NTAzdnJvOGE2Njk3b2M4NjBjcThAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ). For some projects, it will be most efficient to ask everyone's 'regular' weekly availability for the semester rather than updating their availability every week or month. Be clear about what time period you are requesting availability for and who to tell if changes need to be made to the calendar.
+2. Look for blocks of time in which at least two lab assistants are available. These will be the "open" times on the Calendly Calendar in which owners can sign up for appointment slots. 
 
 ### Setting up Calendly Calendar 
 1. Log into the lab [Calendly account](https://calendly.com/) (contact Jeff for login information).
-2. Once on the account dashboard, click on `+ Create` or `New Event Type` to create a new event type for the project then select "One-on-One Event Type".
-3. What event is this?
-  * Use this section to name the event, set a location, give a brief description, and choose
-the display color. The event name is what the participants see, so make it understandable for them but also signals to us what the project is (in case more than one project runs at a time). The location should be In-person meeting and enter 501 Building, room 3. Give a short description to remind the participants what the appointment is.
+2. Once on the account dashboard, find the event called 'Study Template'
+* Click on the gear icon on the upper right corner of the event card
+* Select 'clone' from the menu. This will make a copy of the template event for you to edit.
+3. What event is this? (Click on the section to expand and edit)
+  * Use this section to name the event, set a location, give a brief description, set the url for the event, and choose the display color. The event name is what the participants see, so make it understandable for them but also signals to us what the project is (in case more than one project runs at a time). The location should be In-person meeting and enter 501 Building, room 3. Give a short description to remind the participants what the appointment is. The event link is the URL that will direct people to the booking page for this event. The URL will automatically start with calendly.com/cchil-unl/ so you are just picking an identifier for this specific study to put at the end of the URL.
```

Comment:
```suggestion
  * Use this section to name the event, set a location, give a brief description, set the URL for the event, and choose the display color. The event name is what the participants see, so make it understandable for them but also signals to us what the project is (in case more than one project runs at a time). The location should be In-person meeting and enter 501 Building, room 3. Give a short description to remind the participants what the appointment is. The event link is the URL that will direct people to the booking page for this event. The URL will automatically start with calendly.com/cchil-unl/ so you are just picking an identifier for this specific study to put at the end of the URL.
```